<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>

  <title>Volumetric data</title>
  <meta http-equiv="content-type" content="text/html;charset=utf-8">
  <link rel="stylesheet" type="text/css" href="../css/ww.css">
  <link rel="stylesheet" type="text/css" href="../css/home.css">

  <script type="text/javascript" src="../include/importers.js"></script>
  <script type="text/javascript" src="../include/importWW.js"></script>
  <script type="text/javascript" src="../include/frame1.js"></script>
 


</head>

<body>
  <div id="banner"></div>
  <div class="wwnavbar"></div> 
  <div id="left">
    <div id="menu"></div>
  </div>

  <div id="main">
    <div id="contact">
    Christopher A. Brittin, Steven J. Cook, David H. Hall, Scott W. Emmons, and Netta Cohen. "Volumetric reconstruction of main <i>Caenorhabditis elegans</i> neuropil at two different time points". (2018). Submitted. 
    </div>
    <div id="som">
      <h3>Supporting data</h3>
      <ul>
	<li> Adult data from N2U data set:</li>
	  <ul>
	    <li> <a href="data/n2u_trakem2.tar.gz">TrakEM2 data files </a> (55 MB)</li>
	      <ul>
		<li> n2u_trakem2.xml : <a href="https://imagej.net/TrakEM2">TrakEM2</a> segmentation file.</li>
		<li> n2u_adj.xml : Extracted adjacencies, computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/parsetrakem2/measure_adjacency.py"> measure_adjacency.py </a>.
		<li> n2u_stats.xml : Centroids and area of segments, computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/parsetrakem2/extract_segmentation_stats.py">extract_segmentation_stats.py</a>.
	      </ul> 
	    <li> <a href="data/n2u_trakem2_mipmaps.tar.bz2">TrakEM2 mipmaps files</a> (9.2 GB)</li>
	    <li> <a href="data/adult_segmentation.avi"> Movie</a> that scrolls through the segmented adult electron micrographs (70 MB)</li>
	    <li> <a href="data/n2u_databases.tar.gz"> MySQL databases</a> with adjacency and synaptic connectivity (6.2 MB)</li>
	    <li> <a href="data/n2u_edgelists.tar.gz"> Edgelists</a> for the adjacency and synaptic connectivity (40 KB)</li>
	      <ul>
		<li> 
		  n2u_adjacency.edgelist : Adjacency data in edgelist format <br>
                  Row format: cell1,cell2,total surface area contact between cells in pixels.
		</li>
		<li> 
		  n2u_chemical.edgelist -> Chemical synaptic connections<br>
                  Row format: presynaptic_cell,postsynaptic_cell,synapse weight (EM sections).
		</li>
		<li>
		  n2u_electrical.edgelist -> Electrical synaptic connections<br>
                  Row format: cell1,cell2,synapse weight (EM sections)
		</li>
	      </ul>
	  </ul>
	  <li> L4 data from N2U data set:</li>
          <ul>
            <li> <a href="data/jsh_trakem2.tar.gz">TrakEM2 data files </a> (60 MB)</li>
              <ul>
                <li> jsh_trakem2.xml : <a href="https://imagej.net/TrakEM2">TrakEM2</a> segmentation file.</li>
                <li> jsh_adj.xml : Extracted adjacencies, computed using 
		  <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/parsetrakem2/measure_adjacency.py"> 
		    measure_adjacency.py </a>.
                <li> jsh_stats.xml : Centroids and area of segments, computed using 
		  <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/parsetrakem2/extract_segmentation_stats.py">
		    extract_segmentation_stats.py</a>.
              </ul>
            <li> <a href="data/jsh_trakem2_mipmaps.tar.bz2">TrakEM2 mipmaps files</a> (6.7 GB)</li>
            <li> <a href="data/l4_segmentation.avi"> Movie</a> that scrolls through the segmented adult electron micrographs (86 MB)</li>
            <li> <a href="data/jsh_databases.tar.gz"> MySQL databases</a> with adjacency and synaptic connectivity (3.6 MB)</li>
            <li> <a href="data/jsh_edgelists.tar.gz"> Edgelists</a> for the adjacency and synaptic connectivity (36 KB)</li>
              <ul>
                <li>
                  jsh_adjacency.edgelist : Adjacency data in edgelist format <br>
                  Row format: cell1,cell2,total surface area contact between cells in pixels.
                </li>
                <li>
                  jsh_chemical.edgelist -> Chemical synaptic connections<br>
                  Row format: presynaptic_cell,postsynaptic_cell,synapse weight (EM sections).
                </li>
                <li>
                  jsh_electrical.edgelist -> Electrical synaptic connections<br>
                  Row format: cell1,cell2,synapse weight (EM sections)
                </li>
              </ul>
          </ul>
	  <li><a href="data/cell_adhesion.tar.gz"> Cell adhesion molecule (CAM) expression analysis</a> (860 KB)</li>
	    <ul>
	      <li>wb_ws259.sql : database with gene expression data from <a href="https://wormbase.org">Wormbase</a> version WS259</li>
	      <li>cam_nr_pre_post.csv : list of CAM genes used in the analysis<br>
                Row format: gene name, presynaptic expressed, postsynaptic expressed, number of known isoforms</li>
	      <li>cluster_networks.cys : <a href="https://cytoscape.org">Cytoscape </a> file for displaying expression clusters</li>
	      <li>exp_cluster.gml : Graph file for expression clusters under the Whole-cell Binary Expression model (WBE)</li>
	      <li>exp_clusters_iso.gml -> Graph file for expression clusters under the Isoform Expression model (IE)</li>
	      <li>
		N2U_wbe.csv : Adult LUS scores for the WBE model. 
		Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
		Row format: cell name, presynaptic LUS, gap junction LUS 
	      </li>
	      <li>
		N2U_sbe.csv : Adult LUS scores for the sub-cellular binary expression model (SBE).
                Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
                Row format: cell name, presynaptic LUS, gap junction LUS
              </li>
	      <li>
		N2U_ie.csv : Adult LUS scores for 1000 iterations of IE model. LUS scores are the average across all cells.
                Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
                Row format: average presynaptic LUS score, average gap_junction LUS score.
              </li>
              <li>
                JSH_wbe.csv : L4 LUS scores for the WBE model.
                Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
                Row format: cell name, presynaptic LUS, gap junction LUS
              </li>
              <li>
		JSH_sbe.csv : L4 LUS scores for the sub-cellular binary expression model (SBE).
                Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
                Row format: cell name, presynaptic LUS, gap junction LUS
              </li>
              <li>
                JSH_ie.csv : L4 LUS scores for 1000 iterations of IE model. LUS scores are the average across all cells.
                Computed using <a href="https://github.com/cabrittin/volumetric_analysis/blob/master/cam_analysis/run_cam_models.py">run_cam_models.py</a> <br>
                Row format: average presynaptic LUS score, average gap_junction LUS score.
              </li>
	      <li>aiy_lus.gml : Graph file for LUS score of cell AIY. 
	    </ul>
      </ul>
    </div>
</div>

</body>

</html>
